nitro-node/src/nitro.ts

```diff
@@ -214,23 +217,17 @@ async function runNitroAndLoadModel(
      * The tested threshold is 500ms
      **/
     if (process.platform === "win32") {
-      return await new Promise((resolve) => setTimeout(() => resolve({}), 500));
-    }
-    const spawnResult = await spawnNitroProcess(runMode);
-    if (spawnResult.error) {
-      return spawnResult;
+      await new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
Hello @louis-jan, can you help review this PR? There is a serious bug at this line that I overlooked. The extra `return` makes the library unusable on Windows as the code will return early before spawning nitro process (my bad). 😫

Reply:
Sure @InNoobWeTrust. Thank you